DataSetExplorer/Program.cs

```diff
@@ -38,15 +38,42 @@ static void Main(string[] args)
             //FindInstancesWithAllDisagreeingAnnotationsUseCase();
             //ExportAnnotatedDataSet();
             //CheckAnnotationConsistencyForAnnotator(1);
-            CheckAnnotationConsistencyBetweenAnnotatorsForSeverity(1);
+            //CheckAnnotationConsistencyBetweenAnnotatorsForSeverity(1);
+            CheckMetricsSignificanceInAnnotationsForAnnotator(1);
+        }
+
+        private static void CheckMetricsSignificanceInAnnotationsForAnnotator(int annotatorId)
+        {
+            var instancesGroupedBySmells = GetAnnotatedInstancesGroupedBySmells(annotatorId);
+
+            var exporter = new AnnotationConsistencyByMetricsExporter("D:/ccadet/annotations/sanity_check/anova/Output/");
+            var anovaTest = new AnnotationConsistencyTests.AnovaTest();
+
+            var enumerator = instancesGroupedBySmells.GetEnumerator();
+            while (enumerator.MoveNext())
+            {
+                var codeSmellGroup = enumerator.Current;
+                var codeSmell = codeSmellGroup.Key.Replace(" ", "_");
+
+                exporter.ExportAnnotationsFromAnnotator(annotatorId, codeSmellGroup.ToList(),
```

Comment:
Here you have a minor case of shotgun surgery - if I want to change the path where I want to place the results, I need to change LOC 49, LOC 64 and possibly the body of the ExportAnnotationsFromAnnotator (e.g., if I want to remove sanity check).

Furthermore, if the exporter changes the way it creates the path for storing an excel, this will break the anovaTest.

Come up with a design that centralizes the base path and relative path information.